### PR TITLE
Move if statement out of loop

### PR DESCRIPTION
**Pull request by Leoism**
+ Address comment https://github.com/googleinterns/step155-2020/pull/36#discussion_r450408756 on moving the if statement out of the for loop

### CODE REVIEW

File: leoism/step/portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -66,21 +66,32 @@ function createComment(commentInformation) {
...
+  let nextTenCommentsLoaded = true;
   responsePromise.then((response) => response.json()).then((comments) => {
     for (let i = amtCurrentlyDisplayed; i < nextTenComments; i++) {
+      // Make load more button not appear when all comments have been loaded.
+      if (i === comments.length - 1) {
+        const loadMoreButton = document.getElementById('load-more-button');
+        loadMoreButton.style.display = 'none';
+      }
+
       if (comments[i] === undefined) {
```
Comment (dchin2) [inline leoism/step/portfolio/src/main/webapp/script.js:87]: When would this be undefined? Does being undefined mean that we are at then end of the list?

Comment (Leoism) [reply leoism/step/portfolio/src/main/webapp/script.js:87]: The for loop iterates 10 times every time. If the list is short of ten, say the list only has 4 elements, then comments[i] would be undefined at the 5th iteration. At this point, we know we iterated i times so the data-display attribute is updated to i

Comment (Leoism) [reply leoism/step/portfolio/src/main/webapp/script.js:87]: I added a comment to explain this

**Review by dchin2 — COMMENTED** [review]
nice

**Review by dchin2 — APPROVED** [review]

**Pull request merged**